internal/output/verbosity/enum.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package verbosity
+
+type Enum int
```

Comment:
>Enum [](http://example.com/codeflow?start=5&length=4)

how about `Level` or even `Verbosity` ? `Enum` is the name of the mechanism, not the semantic of the type.

Reply:
I get what you're saying, but I've done it this way so it looks like this when using it:

 output.New(verbosity.Enum(options.LoggingLevel))

golang uses the package name to add the context. 

But agree this might be me trying to make it familiar to a C# user.

I'll change it to verbosity.Level